src/config.ts

```diff
@@ -51,27 +51,18 @@ export default {
   TIMEZONES: [
     //AMERICA
 
-    { abbr: 'HST', zone: 'Pacific/Honolulu' },
```

Comment:
Seems brief enough, but perhaps some better known locations could be used?

Reply:
Yah perhaps.
But, as far as I am aware there aren't a lot of ppl from Hawaiian Aleutian Timezone in our organisation
Perhaps, we can ask @eddiejaoude regarding which ones should be added/removed ?

Reply:
Oops, I meant to open the review on the code block, but it seems to have settled on just one line by accident... but I was referring to the discord community at large

Reply:
Timezones like UTC and UK, and CEST and EET, fall under the same timezone. We can remove one of each pair. On the other hand, we might have skipped some timezones entirely.

Reply:
agreed(I kept uk as most ppl use uk while mentioning livestream timings etc.)
CEST and EET are weird(they are same for half the year and change, I think)